index.html

```diff
@@ -1688,40 +1709,75 @@
             </dd>
           </dl>
           <ol>
-            <li>If the <a>presentation connection state</a> of
-            <code>presentationConnection</code> is not <code>connected</code>
-            or <code>connecting</code>, then abort these steps.
-            </li>
-            <li>Set <a>presentation connection state</a> of
+            <li>Start to tear down the underlying connection in a non abrupt
+            manner. The mechanism by which this is achieved is
+            implementation-specific but should signal the intention to tear
+            down the connection to the <a>destination browsing context</a> and
+            eventually render the connection <a link-for=
+            "PresentationConnection">closed</a>.
+            </li>
+            <li>If the procedure was initiated by the <code><a link-for=
```

Comment:
Maybe the "start closing" operation should take the arguments of the "closing" operation so we don't need the conditional phrasing here.